connector/src/main/java/org/geysermc/connector/command/defaults/DumpCommand.java

```diff
@@ -74,7 +75,10 @@ public void execute(GeyserSession session, CommandSender sender, String[] args)
                     case "offline":
                         offlineDump = true;
                         break;
-
+                    case "logs":
```

Comment:
My opinion here is that we stick to one option here (remove the `log` case), as no one is going to know about that as an option unless we say it, and then we have two redundant switches floating around. Open to discussion as Konica brought this up.

Reply:
Honestly I'm indifferent, the majority of people are only going to know the subcommand if we ask them to run it